api/Animation.json

```diff
@@ -0,0 +1,945 @@
+{
+  "api": {
+    "Animation": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Animation",
+        "support": {
+          "webview_android": {
+            "version_added": "39",
+            "notes": "Before Chrome 44, implemented as <code>AnimationPlayer</code> (interface name in an early version of the spec)."
+          },
+          "chrome": {
+            "version_added": "39",
+            "notes": "Before Chrome 44, implemented as <code>AnimationPlayer</code> (interface name in an early version of the spec)."
+          },
+          "chrome_android": {
+            "version_added": "39",
+            "notes": "Before Chrome 44, implemented as <code>AnimationPlayer</code> (interface name in an early version of the spec)."
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "48"
+          },
+          "firefox_android": {
+            "version_added": "48"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "26"
+          },
+          "opera_android": {
+            "version_added": "26"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "Animation": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Animation/Animation",
+          "description": "<code>Animation()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": "61"
+            },
+            "chrome": {
+              "version_added": "61"
+            },
+            "chrome_android": {
+              "version_added": "61"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "48",
+              "notes": "The Web Animations API is only enabled by default in Firefox Developer Edition and Nightly builds. You can enable it in release builds by setting the preference <code>dom.animations-api.core.enabled</code> to <code>true</code>, and can disable it in any Firefox version by setting this preference to <code>false</code>."
```

Comment:
This has to be described using the "flags", the schema documentation has examples.

Reply:
I'll do this all in a bit - thank you! 😄